react-backend/client/package.json

```diff
@@ -5,18 +5,20 @@
   "dependencies": {
     "axios": "^0.18.0",
     "babel-eslint": "^8.2.2",
-    "chart.js": "^1.1.1",
+    "chart.js": "^2.7.2",
     "normalize.css": "^8.0.0",
     "polished": "^1.9.2",
-    "prop-types": "^15.6.0",
+    "chart.js": "^2.7.2",
```

Comment:
You've got a duplicate of `chart.js` here.

Reply:
Oops, fixed.